source/patterns/@aws-solutions-constructs/core/test/cloudfront-distribution-s3-helper.test.ts

```diff
@@ -315,6 +315,65 @@ test('test cloudfront override cloudfront logging bucket ', () => {
   });
 });
 
+test('test cloudfront disable cloudfront logging', () => {
+  const stack = new Stack();
+  const [sourceBucket] = buildS3Bucket(stack, {});
+
+  CloudFrontDistributionForS3(stack, sourceBucket, {
+    enableLogging: false,
+  });
+
+  expect(stack).toHaveResourceLike("AWS::CloudFront::Distribution", {
```

Comment:
This does not explicitly test for the absence of `Logging` prop which would be better, is there a test checking for absence elsewhere in the codebase?

Reply:
I believe this command looks for an identical resource, so if a logging prop exists it will fail. What we are missing (and what would have prevented this issue) is a check that the logging bucket does not exist. You could add that with an `expect(stack).not.toHaveResourceLike("AWS::S3::Bucket", { _define logging bucket here_ });` call if you're willing to add that...